libr/anal/fcn.c

```diff
@@ -1544,22 +1544,23 @@ R_API int r_anal_fcn_cc(RAnal *anal, RAnalFunction *fcn) {
 	return result;
 }
 
-R_API char *r_anal_fcn_to_string(RAnal *a, RAnalFunction *fcn) {
+R_API char *r_anal_function_get_name(RAnalFunction *function) {
+	RAnal *a = function->anal;
 	char *ret = NULL, *arg_i = NULL, *args = strdup ("");
 	// RList *args_list = r_list_newf ((RListFree) free);
 	unsigned int argc = 0, i, j;
 	const char *ret_type = sdb_const_get (a->sdb_types,
-	  r_str_newf ("func.%s.ret", fcn->name), 0);
+	  r_str_newf ("func.%s.ret", function->name), 0);
 	const char *argc_str = sdb_const_get (a->sdb_types,
-	  r_str_newf ("func.%s.args", fcn->name), 0);
+	  r_str_newf ("func.%s.args", function->name), 0);
```

Comment:
Tab vs spaces?

Reply:
this  is leaking, nobody is freeing this r_str_newf, you can use sdb_fmt too. maybe we should add sdb_const_getf() that takes a format string as argument

Reply:
feedback on this @ret2libc ?

Reply:
ok either way for me

Reply:
Changed it since we also changed function name to `r_anal_function_get_name`

Reply:
> this is leaking, nobody is freeing this r_str_newf, you can use sdb_fmt too. maybe we should add sdb_const_getf() that takes a format string as argument

This would be great imho